[v17] docs: update outdated Access List and integration web ui

ptgott opened the pull request:
Backports #57868

* updating ui steps and images

* re-added and modified ui images per Roman's feedback restore formatting for deleting integration section

* replacing blurry image per feedback

* line space linting error correction

* add missing image references

* shortening the lines of chars

* fixed typo

* removing this from the pr as this page has since been moved and corrected in main, and is causing a conflict here

mmcallister committed:
docs: update outdated Access List and integration web ui

Backports #57868

* updating ui steps and images

* re-added and modified ui images per Roman's feedback
restore formatting for deleting integration section

* replacing blurry image per feedback

* line space linting error correction

* add missing image references

* shortening the lines of chars

* fixed typo

* removing this from the pr as this page has since been moved and corrected in main, and is causing a conflict here

mmcallister reviewed (approved):
Thanks for unearthing this one!

ptgott committed:
Correct Okta integration enrolment page

- Revert a change that added an incorrect image and instructions to
  visit the Identity Security tab.
- Correct instructions to visit the Access tab to visit the Zero Trust
  Access tab.

ptgott commented:
Need to add this PR to this backport before we merge:
- https://github.com/gravitational/teleport/pull/59595
